lib/index.js

```diff
@@ -103,12 +128,32 @@ export const createStore = (reducer, initialState, enhancer) => {
  * @returns {Function} - A function that expects two arguments: an Object of key/value pairs where the values are
  *                       Backbone entities, and an optional store enhancer. Returns a store.
  */
-export const bbCreateStore = (storeCreator = createStore) => (backboneEntities, enhancer = noop) => {
-    const combinedBBEntities = bbCombineEntities(backboneEntities);
+export const bbCreateStore = (storeCreator = createStore) => (bbEntityOrReducer, enhancer = noop) => {
+    let bbReturningReducer;
+    let combinedBBEntities;
+    let passedBBEntity = typeof bbEntityOrReducer !== 'function';
```

Comment:
I totally agree that what you described has a lot of value. I was actually advocating in favor of the "reducer" API style, not the opposite.

I think my point is that I'm not convinced having BOTH API usages bring enough value to justify them.
For instance, if you use the "entities hash" API style, you would most likely still have to modify some backbone code to make things work: you have to add the event listeners to your model/collection that will modify the model/colelction state when triggered ([example from our sample](https://github.com/adamterlson/backbone-redux-store/blob/master/sample/src/CounterModel.js#L5-L9)). So why would that use case be considered more basic than the "reducer" style?

Hope that makes sense!